src/otx/core/model/detection_3d.py

```diff
@@ -295,3 +216,349 @@ def get_classification_layers(self, prefix: str = "model.") -> dict[str, dict[st
         return classification_layers
+
+
+class MonoDETRModel(ImageModel):
+    """A wrapper for MonoDETR 3d object detection model."""
+
+    __model__ = "mono_3d_det"
+
+    def __init__(self, inference_adapter: InferenceAdapter, configuration: dict[str, Any], preload: bool = False):
+        """Initializes a 3d detection model.
+
+        Args:
+            inference_adapter (InferenceAdapter): inference adapter containing the underlying model.
+            configuration (dict, optional): configuration overrides the model parameters (see parameters() method).
+              Defaults to dict().
+            preload (bool, optional): forces inference adapter to load the model. Defaults to False.
+        """
+        super().__init__(inference_adapter, configuration, preload)
+        self._check_io_number(3, 5)
+
+    def preprocess(self, inputs: dict[str, np.ndarray]) -> tuple[dict[str, Any], ...]:
+        """Preprocesses the input data for the model.
+
+        Args:
+            inputs (dict[str, np.ndarray]): a dict with image, calibration matrix, and image size
+
+        Returns:
+            tuple[dict[str, Any], ...]: a tuple with the preprocessed inputs and meta information
+        """
+        return {
+            self.image_blob_name: inputs["image"][None],
+            "calib_matrix": inputs["calib"],
+            "img_sizes": inputs["img_size"][None],
+        }, {
+            "original_shape": inputs["image"].shape,
+            "resized_shape": (self.w, self.h, self.c),
+        }
+
+    def _get_inputs(self) -> tuple[list[Any], list[Any]]:
+        """Defines the model inputs for images and additional info.
+
+        Raises:
+            WrapperError: if the wrapper failed to define appropriate inputs for images
+
+        Returns:
+            - list of inputs names for images
+            - list of inputs names for additional info
+        """
+        image_blob_names, image_info_blob_names = [], []
+        for name, metadata in self.inputs.items():
+            if len(metadata.shape) == 4:
+                image_blob_names.append(name)
+            elif len(metadata.shape) == 2:
+                image_info_blob_names.append(name)
+
+        if not image_blob_names:
+            self.raise_error(
+                "Failed to identify the input for the image: no 4D input layer found",
+            )
+        return image_blob_names, image_info_blob_names
+
+    def postprocess(
+        self,
+        outputs: dict[str, np.ndarray],
+        meta: dict[str, Any],
+    ) -> dict[str, Any]:
+        """Applies SCC decoded to the model outputs.
+
+        Args:
+            outputs (dict[str, np.ndarray]): raw outputs of the model
+            meta (dict[str, Any]): meta information about the input data
+
+        Returns:
+            dict[str, Any]: postprocessed model outputs
+        """
+        result = {}
+        for k in outputs:
+            result[k] = np.copy(outputs[k])
+
+        return result
```

Comment:
```suggestion
        return result
```
Isn't change above possible?

Reply:
No, it's not. Outputs are owned by infer request.